lib/render/editor/editor.html.erb

```diff
@@ -946,9 +978,12 @@
 
       _setEditorToolbox: function () {
         const editor = $("mu-gobstones-custom-editor")[0];
-        $.get(this.toolboxUrl, function (toolboxXml) {
-          editor.setToolbox(toolboxXml);
-        });
+
+        // editor may have not yet been loaded
+        // so there is nothing to do yet
+        if (!editor) return;
```

Comment:
Calling this inside Polymer's `async` function _should_ guarantee all siblings (including the editor) are loaded. Did you find a scenario where the editor is not loaded?

Reply:
Assuming such scenario exists, not loading the toolbox would lead to an inconsistency, because this code would never be called again. :thinking:

Reply:
The weird thing is that it happens (see the screenshots above) and that it is called multiple times, because after the first too-early call, then the code is properly initialized and works :shrug:

Reply:
I will check the async function anyway

Reply:
I agree with both of you that that should not happen. Let's try to track why it does happen. I'm pretty sure that it is something related to something specific about classroom or angular.
The rest is great :100: :+1:

Reply:
If added a debug line. However, I can confirm that this happens everytime in classrom except for the first load. I also confirm that the code is called exactly once - as @faloi said.